src/Sylius/Bundle/AdminBundle/Tests/Command/CreateAdminUserCommandTest.php

```diff
@@ -0,0 +1,156 @@
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Sylius\Component\User\Canonicalizer\CanonicalizerInterface;
+use Sylius\Component\User\Repository\UserRepositoryInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Tester\CommandTester;
+
+final class CreateAdminUserCommandTest extends TestCase
+{
+    private MockObject $userRepository;
+
+    private MockObject $factory;
+
+    private MockObject $canonicalizer;
+
+    private CommandTester $command;
+
+    /** @test */
+    public function it_creates_an_admin_user_if_accepted_in_the_summary(): void
+    {
+        $this->canonicalizer
+            ->method('canonicalize')
+            ->with('SYLius@exaMPLE.com')
+            ->willReturn('sylius@example.com')
+        ;
+
+        $this->userRepository
+            ->method('findOneByEmail')
+            ->with('sylius@example.com')
+            ->willReturn(null)
+        ;
+
+        $this->factory->method('createNew')->willReturn(new AdminUser());
+
+        $this->command->setInputs([
+            'email' => 'SYLius@exaMPLE.com',
+            'username' => 'Sylius',
+            'firstname' => 'Sylius',
+            'lastname' => 'Admin',
+            'password' => 'sylius',
+            'local_code' => 'en_US',
+            'admin_user_enabled' => 'yes',
+            'creation_confirmation' => 'yes',
+        ]);
+
+        self::assertEquals(Command::SUCCESS, $this->command->execute([]));
+    }
+
+    /** @test */
+    public function it_does_not_create_an_admin_user_if_declined_in_the_summary(): void
+    {
+        $this->canonicalizer
+            ->method('canonicalize')
+            ->with('SYLius@exaMPLE.com')
+            ->willReturn('sylius@example.com')
+        ;
+
+        $this->userRepository
+            ->method('findOneByEmail')
+            ->with('sylius@example.com')
+            ->willReturn(null)
+        ;
+
+        $this->factory->method('createNew')->willReturn(new AdminUser());
+
+        $this->command->setInputs([
+            'email' => 'SYLius@exaMPLE.com',
+            'username' => 'Sylius',
+            'firstname' => 'Sylius',
+            'lastname' => 'Admin',
+            'password' => 'sylius',
+            'local_code' => 'en_US',
+            'admin_user_enabled' => 'yes',
+            'creation_confirmation' => 'no',
+        ]);
+
+        self::assertEquals(Command::INVALID, $this->command->execute([]));
+    }
+
+    /** @test */
+    public function it_does_not_create_an_admin_user_if_user_already_exists(): void
+    {
+        $this->canonicalizer
+            ->method('canonicalize')
+            ->with('SYLius@exaMPLE.com')
+            ->willReturn('sylius@example.com')
+        ;
+
+        $this->userRepository
+            ->method('findOneByEmail')
+            ->with('sylius@example.com')
+            ->willReturn(new AdminUser())
+        ;
+
+        $this->command->setInputs(['Email' => 'SYLius@exaMPLE.com']);
```

Comment:
```suggestion
        $this->command->setInputs(['email' => 'SYLius@exaMPLE.com']);
```
For consistency's sake